packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/repository.test.ts

```diff
@@ -3792,7 +3792,6 @@ describe('SavedObjectsRepository', () => {
                 'updated_at',
                 'created_at',
                 'originId',
-                'title',
```

Comment:
See previous comment (on the v5 field compat) for the reason of this test change